project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -235,3 +229,36 @@ function c4m_user_notifications_notify_inactive_users() {
  */
 function c4m_user_notifications_notify_comment_on_content($comment) {
 }
+
+/**
+ * Email notification to group requester, that group is waiting for review.
+ */
+function c4m_user_notifications_notify_new_group_pending($node) {
```

Comment:
Not sure what you mean.
Here we have empty function, as part of infrastructure.
There we have it's implementation.